indico/modules/events/registration/templates/management/regform_reglist.html

```diff
@@ -23,35 +23,57 @@
             <div class="group left">
                 <button class="i-button icon-settings js-dialog-action"
                    data-href="{{ url_for('.customize_reglist', regform) }}"
-                   data-title="{% trans %}Customize report{% endtrans %}">
+                   data-title="{% trans %}Registration list configuration{% endtrans %}">
                     {% trans %}Customize report{% endtrans %}
                 </button>
             </div>
             <div class="group left">
-                <a href="#" class="i-button icon-plus">{% trans %}Add new{% endtrans %}</a>
-                <a href="#" class="i-button icon-remove">{% trans %}Remove{% endtrans %}</a>
+                <a href="{{ url_for('.create_registration', regform) }}" class="i-button icon-plus">{% trans %}Add new{% endtrans %}</a>
+                <a class="i-button icon-remove js-requires-selected-row disabled js-delete-registrations"
+                   data-href="{{ url_for('.delete_registrations', regform) }}"
+                   data-method="POST"
+                   data-confirm="{% trans %}Do you really want to delete the selected registrations?{% endtrans %}">
+                    {%- trans %}Remove{% endtrans -%}
+                </a>
             </div>
             <div class="group left">
-                <a href="#" class="i-button arrow button" data-toggle="dropdown">{% trans %}Export{% endtrans %}</a>
+                <a class="i-button arrow button js-requires-selected-row disabled" data-toggle="dropdown">
+                    {%- trans %}Export{% endtrans -%}
+                </a>
                 <ul class="dropdown">
-                    <li><a href="#" class="icon-file-pdf">PDF</a></li>
-                    <li><a href="#" class="icon-file-excel">CSV</a></li>
+                    <li>
+                        <a href="#" class="icon-file-pdf js-requires-selected-row disabled js-pdf-export"
```

Comment:
I figured that without the href I would have to add some css to make it look the same (it looses it's blue colour by removing the href)